lib/mhv_logging/api/audits.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure I understand why this module is separate from the Sessions one or really why they're modules at all rather than being inside client.rb directly. Seems like a strategy to factor the code out, but if it's not being reused then I think we're adding complexity with little gain.